box2d/src/test/kotlin/ktx/box2d/worldsTest.kt

```diff
@@ -218,4 +218,39 @@ class WorldsTest : Box2DTest() {
     assertEquals(0f, RayCast.TERMINATE, tolerance)
     assertEquals(-1f, RayCast.IGNORE, tolerance)
   }
+
+  @Test
+  fun `should query for overlapping fixtures with AABB and stop`() {
+    val world = createWorld()
+    val matchingEdge1 = world.body {}.edge(from = Vector2.Zero, to = Vector2(0f, 2f)) {}
+    val matchingEdge2 = world.body {}.edge(from = Vector2(1f, 0f), to = Vector2(1f, 2f)) {}
+    val matchedFixtures = mutableSetOf<Fixture>()
+
+    world.query(-1f, 1f, 1f, 1f) { fixture ->
```

Comment:
Use named parameters. Tests showcase the API, 4 unnamed floats can be somewhat confusing.